Functions.Templates/ProjectTemplates.nuspec

```diff
@@ -1,13 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
-    <id>Microsoft.AzureFunctions.ProjectTemplates</id>
+    <id>Microsoft.Azure.Functions.ProjectTemplates</id>
     <version>1.0.0</version>
     <authors>Microsoft</authors>
     <description>Azure Functions Project Template Pack for Microsoft Template Engine</description>
     <language>en-US</language>
     <projectUrl>https://go.microsoft.com/fwlink/?linkid=848145</projectUrl>
     <licenseUrl>https://raw.githubusercontent.com/Azure/azure-webjobs-sdk-templates/8ba7a6f1f907edb5de30c1ef4791d97711d84cb6/LICENSE.txt</licenseUrl>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>

```

Comment:
Use the aka.ms link used by the other projects here.

Reply:
done.